airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -33,22 +33,33 @@
     from airflow.models import TaskInstance
 
 
```

Comment:
@bolkedebruin - Does the following method step on your serializing PR any?

Reply:
I do not think so, but I am not familiar with this code.